reading/git.livemd

```diff
@@ -20,7 +21,7 @@ Mix.install([
 
 ## Setup
 
-Ensure you run the `ea` command to evaluate all Elixir cells before starting. Alternatively,
+Ensure you type the `ea` keyboard shortcut to evaluate all Elixir cells before starting. Alternatively,
```

Comment:
I'd like to ensure consistency between all notebooks that have this section. I'll merge this in because I agree it's an improvement! We should modify all setup sections to match it. I'll make that change sometime today.